Automattic/jetpack
# Inline PDFs: simplify the output of PDF embeds.

#### Changes proposed in this Pull Request:

﻿* This commit syncs r237795-wpcom.
* Differential revision: D72439-code.
* Internal reference: p3btAN-1Ab-p2

Core now includes a File Block that can be used to embed PDFs on your site, from your media library. Let's stop inline PDF embeds from Jetpack to let Core do the work.

**Note**: One of the differences between Core's and Jetpack's implementation is that our implementation supports embeds of files from other sources.

This fixes a security issue, as removing the embeds avoids the potential of having a malicious PDF from a third-party site that would run arbitrary code.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

1. Go to Posts > Add New
2. Paste a PDF such as `https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf` on its own line.
3. Publish post.
4. On the frontend, you should see a link to that PDF, instead of an embed.
